src/pages/komodo-defi-framework/api/v20/my_recent_swaps/index.mdx

```diff
@@ -0,0 +1,390 @@
+                  }
+                },
+                "type": "TakerPaymentSpent"
+              },
+              "timestamp": 1561483355081
+            },
+            {
+              "event": {
+                "data": {
+                  "tx_hash": "858f07d0a4e74318497a6e3ff4d7b68b60ad21b5c8e90b9b485f0ddaed71d0dc",
+                  "tx_hex": "0400008085202f8901f47216323973b32881394a18c4e77581ce794dad1033854b9d3e5e7890c836ba00000000d8483045022100847a65faed4bea33c5cbccff2bee7c1292871a3b130bd2f23e696bd80c07365f02202039ea02b4463afd4f1e2b20b348d64b40aaea165f8dfb483293e2b368d536fe012066ed6c24bbb4892634eac4ce1e1ad0627d6379da4443b8d656b64d49ef2aa7a3004c6b6304af46135db1752102631dcf1d4b1b693aa8c2751afc68e4794b1e5996566cfc701a663f8b7bbbe640ac6782012088a914ba5128bcca5a2f7d2310054fb8ec51b80f352ef3882102031d4256c4bc9f99ac88bf3dba21773132281f65f9bf23a59928bce08961e2f3ac68ffffffff01583e0f00000000001976a91405aab5342166f8594baf17a7d9bef5d56744332788ac4b4a125d000000000000000000000000000000"
+                },
+                "type": "MakerPaymentSpent"
+              },
+              "timestamp": 1561483358319
+            },
+            {
+              "event": {
+                "type": "Finished"
+              },
+              "timestamp": 1561483358321
+            }
+          ],
+          "my_info": {
+            "my_amount": "0.01",
+            "my_coin": "BCH",
+            "other_amount": "0.01",
+            "other_coin": "BEER",
+            "started_at": 1561481967
+          },
+          "maker_coin": "BEER",
+          "maker_amount": "0.01",
+          "taker_coin": "BCH",
+          "taker_amount": "0.01",
+          "gui": null,
+          "mm_version": "unknown",
+          "success_events": [
+            "Started",
+            "Negotiated",
+            "TakerFeeSent",
+            "MakerPaymentReceived",
+            "MakerPaymentWaitConfirmStarted",
+            "MakerPaymentValidatedAndConfirmed",
+            "TakerPaymentSent",
+            "TakerPaymentSpent",
+            "MakerPaymentSpent",
+            "Finished"
+          ],
+          "type": "Taker",
+          "uuid": "491df802-43c3-4c73-85ef-1c4c49315ac6"
+        }
+      ]
+    }
+  }
+  ```
+
+  Response (error)
+
+  ```json
+  {
+    "error": "lp_swap:1454] from_uuid e299c6ece7a7ddc42444eda64d46b163eaa992da65ce6de24eb812d715184e41 swap is not found"
+  }
+  ```
+</CollapsibleSection>
```

Comment:
I believe this error should be `from_uuid e299c6ece7a7ddc42444eda64d46b163eaa992da65ce6de24eb812d715184e41 is not found` , the word `swap` is not in the error.

Reply:
I am unable to replicate this error. Currently, using `"from_uuid": "e299c6ece7a7ddc42444eda64d46b163eaa992da65ce6de24eb812d715184e40"` (which does not exist) still returns other recent swaps.

Reply:
Right, it doesn't return an error, it ignores the uuid as if it's not there and just returns the results from the start or row 0. This should be the case for legacy/v1 RPC as well, please check this as this error is in one of the responses in the v1 docs as well.

Reply:
I've removed that error from the v2 doc, and updated the v1 doc with a couple of current error examples. Using a non-existent uuid (in the correct format) returns

```json
{
    "error": "rpc:215] dispatcher_legacy:141] lp_swap:1299] my_swaps_storage:102] Unknown SQL error: Query returned no rows"
}
```